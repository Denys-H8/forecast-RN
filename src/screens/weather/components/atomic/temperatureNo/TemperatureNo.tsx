import React from 'react';
import { View } from 'react-native';

import CustomText from 'src/components/text';
import { styles } from './temperatureNo.styles';

interface IProps {
  temp: number;
}

const TemperatureNo: React.FC<IProps> = ({ temp }) => (
  <View style={styles.container}>
    <CustomText style={styles.number}>{temp}</CustomText>
    <CustomText style={styles.celsius}>0</CustomText>
  </View>
);

export default TemperatureNo;
