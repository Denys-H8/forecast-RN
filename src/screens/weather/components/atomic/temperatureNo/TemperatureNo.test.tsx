import React from 'react';
import { render } from '@testing-library/react-native';
import TemperatureNo from './TemperatureNo';

const temp = 19;

describe('TemperatureNo ', () => {
  test('should render TemperatureNo', () => {
    const { toJSON } = render(<TemperatureNo temp={temp} />);

    expect(toJSON()).toMatchSnapshot();
  });
});
