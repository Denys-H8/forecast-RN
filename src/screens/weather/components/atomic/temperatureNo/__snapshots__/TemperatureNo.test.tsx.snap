// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TemperatureNo  should render TemperatureNo 1`] = `
<View
  style={
    Object {
      "alignItems": "flex-start",
      "flexDirection": "row",
    }
  }
>
  <Text
    style={
      Array [
        Object {
          "fontFamily": "WorkSans-Regular",
        },
        Object {
          "color": "#fff",
          "fontSize": 60,
          "lineHeight": 70,
        },
      ]
    }
  >
    19
  </Text>
  <Text
    style={
      Array [
        Object {
          "fontFamily": "WorkSans-Regular",
        },
        Object {
          "color": "#fff",
          "fontSize": 25,
          "lineHeight": 30,
        },
      ]
    }
  >
    0
  </Text>
</View>
`;
